features/htcondor/client/ce.pan

```diff
@@ -1,51 +1,64 @@
 unique template features/htcondor/client/ce;
 
-variable CONDOR_CONFIG={
-
-  if (FULL_HOSTNAME == LRMS_SERVER_HOST) {
-    file_list=list('submit');
-  } else {
-    file_list=list('params', 'submit')
-  };
-
-  foreach(i;file;file_list){
-    num = length( SELF['cfgfiles']);
-    SELF['cfgfiles'][num] = nlist( 'name',file,'contents','features/htcondor/templ/' + file);
-  };
-
-  if(!is_defined(SELF['options']['submit'])){
-    SELF['options']['submit'] = nlist();
-  };
-
-  SELF;
+variable CONDOR_CONFIG = {
+    if (FULL_HOSTNAME == LRMS_SERVER_HOST) {
+        file_list = list('submit');
+    } else {
+        file_list = list('params', 'submit')
+    };
+
+    foreach (i; file; file_list) {
+        num = length( SELF['cfgfiles']);
+        SELF['cfgfiles'][num] = dict( 'name', file, 'contents', 'features/htcondor/templ/' + file);
+    };
+
+    if (!is_defined(SELF['options']['submit'])) {
+        SELF['options']['submit'] = dict();
+    };
+
+    SELF;
 };
 
 
-include {'features/htcondor/server/groups'};
-include {'features/htcondor/client/policies'};
+include 'features/htcondor/server/groups';
+include 'features/htcondor/client/policies';
 
-variable BLPARSER_WITH_UPDATER_NOTIFIER=true;
+variable BLPARSER_WITH_UPDATER_NOTIFIER = true;
 
-variable CE_QUEUES ?={
-  SELF['vos']['default']=VOS;
-  SELF;
+variable CE_QUEUES ?= {
+    SELF['vos']['default'] = VOS;
+    SELF;
```

Comment:
This is just a default right?, so preserving the left-hand-side is not a concern and this could be written as:
```pan
variable CE_QUEUES ?= dict(
    'vos', dict(
        'default', VOS,
    ),
);
```